lib/lm_docstache/block.rb

```diff
@@ -1,71 +0,0 @@
-module LMDocstache
-  class Block
-    attr_reader :name, :opening_element, :content_elements, :closing_element, :inverted, :condition, :inline
-    def initialize(name:, data:, opening_element:, content_elements:, closing_element:, inverted:, condition: nil, inline: false)
-      @name = name
-      @data = data
-      @opening_element = opening_element
-      @content_elements = content_elements
-      @closing_element = closing_element
-      @inverted = inverted
-      @condition = condition
-      @inline = inline
-    end
-
-    def type
-      @type ||= if @inverted
-        :conditional
-      else
-        if @data.get(@name).is_a? Array
-          :loop
-        else
-          :conditional
-        end
-      end
-    end
-
-    def loop?
```

Comment:
talked about this, but we can't just remove this functionality without recreating it elsewhere :p

Reply:
As we spoke on private in Slack, we'll drop support for it now, and come up with a new solution for that in upcoming versions.